server/index.js

```diff
@@ -179,6 +210,46 @@ app.post('/api/trips', uploadsMiddleware, (req, res, next) => {
     });
 });
 
+app.post('/api/trips/comments/:tripId', (req, res, next) => {
+  const tripId = Number(req.params.tripId);
+  const { userId } = req.user;
+  if (!Number.isInteger(tripId) || tripId < 1) {
+    res.status(400).json({
+      error: 'tripId must be a positive integer'
+    });
+    return;
+  }
+  const { content } = req.body;
+  if (!content) {
+    res.status(400).json({
+      error: 'missing data'
+    });
```

Comment:
This error message would be more helpful if it said `"missing content"` instead of `"missing data"`.